src/pages/quiz/QuizDisplay.jsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import { shuffle, Timer } from "../../utils/util-index";
+import { QuestionsDisplay } from "./QuestionsDisplay";
+import { BiTimer } from "react-icons/bi";
+import { GrNext, GrPrevious } from "react-icons/gr";
+
+export const QuizDisplay = (props) => {
+  const {
+    questionCount,
+    questions,
+    questionDecrement,
+    questionIncrement,
+    submitHandler,
+  } = props;
+  return (
+    <>
+      <div className="flex justify-center flex-col items-center">
+        <div className="bg-gray-900 mt-16 rounded-lg mb-10 h-340 w-4/5 ">
+          <div className="flex justify-between text-yellow-300 px-12 py-4">
+            <div className="flex items-center gap-2 justify-center">
+              <span>
+                <BiTimer />
+              </span>
+              <span>
+                <Timer />
+              </span>
+            </div>
+            <div>{`${questionCount}/10`}</div>
+            <div className="italic">point 5</div>
+          </div>
+          <div className="flex h-full px-12 py-4 w-full">
+            {questions &&
+              questions.map(
+                ({ question, correct_answer, incorrect_answers, alphaKey }) => {
+                  const options = incorrect_answers.concat(correct_answer);
+                  shuffle(options);
+                  return (
+                    <QuestionsDisplay
+                      key={question}
+                      question={question}
+                      alphaKey={alphaKey}
+                      options={options}
+                      questionCount={questionCount}
```

Comment:
instead of expanding it two times here, you could directly take questionData inside map and in the component
you can do <QuestionsDisplay key={} {...questionData} />